pkg/antctl/raw/multicluster/add/member_cluster.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package add
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	multiclusterv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
+	"antrea.io/antrea/pkg/antctl/raw"
+	multiclusterscheme "antrea.io/antrea/pkg/antctl/raw/multicluster/scheme"
+)
+
+type memberClusterOptions struct {
+	namespace      string
+	clusterSet     string
+	serviceAccount string
+}
+
+var memberClusterOpt *memberClusterOptions
+
+var memberClusterExamples = strings.Trim(`
+Add a new member cluster to a ClusterSet
+$ antctl mc add member-cluster <CLUSTER_ID> -n <NAMESPACE> --clusterset <CLUSTERSET> --service-account <SERVICE_ACCOUNT>
+`, "\n")
+
+func (o *memberClusterOptions) validateAndComplete() error {
+	if o.namespace == "" {
+		return fmt.Errorf("the namespace cannot be empty")
```

Comment:
You used "Namespace", "ClusterSet", "ServiceAccount" in other commands. Please be consistent. Could you check and update all commands to use the same format?

Reply:
Done. Changed all the resources to uppercase  such as ServiceAccount